main.star

```diff
@@ -11,6 +11,7 @@ def run(
     grafana_version="9.5.12",
     grafana_alerting_template="",
```

Comment:
I realized that passing in an empty string for `grafana_alerting_template` won't let the package run because `read_file` doesn't accept empty strings. Would you mind adding a check for this so that emtpy defaults still allow the package to run? eg
```
if grafana_alerting_plate != "":
     <add to files artifact config>
```